### PR TITLE
[WIP] Upgrade VS Code Metals, the Scala language client, to version 1.9.5

### PR DESCRIPTION
**Pull request by sideeffffect**
Signed-off-by: sideeffffect <ondra.pelech@gmail.com>

### CODE REVIEW

File: v3/plugins/scalameta/metals/1.9.5/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://raw.githubusercontent.com/scalameta/metals-vscode/v1.9.5/images/logo.png
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-11-02"
+spec:
+  containers:
+    - image: "quay.io/gattytto/scala090:0.9.0"
```
Comment (ericwill) [inline v3/plugins/scalameta/metals/1.9.5/meta.yaml:15]: What's different about this image? It's probably best to update the side in che-sidecar-scala, no?

Comment (sideeffffect) [reply v3/plugins/scalameta/metals/1.9.5/meta.yaml:15]: I've created a ticket to fix the image https://github.com/che-dockerfiles/che-sidecar-scala/issues/10

Comment (gattytto) [reply v3/plugins/scalameta/metals/1.9.5/meta.yaml:15]: when [this one](https://github.com/eclipse/che-plugin-registry/pull/682/files) is merged you may use the new sidecar ID for quay.io/eclipse/che-sidecar-scala:0.9.0-xyz here @sideeffffect

File: vscode-extensions.json [hunk 2]
```diff
@@ -344,7 +344,7 @@
     },
     {
       "repository": "https://github.com/scalameta/metals-vscode",
-      "revision": "v1.9.4",
+      "revision": "v1.9.5",
       "sidecar": {
         "image": "quay.io/eclipse/che-sidecar-scala:0.9.0-b92a5f9",
```
Comment (gattytto) [inline vscode-extensions.json:349]: here you will need to use the new ID after https://github.com/eclipse/che-plugin-registry/pull/682/files is merged @sideeffffect

**Comment by sideeffffect** [comment]
deprecated by https://github.com/eclipse/che-plugin-registry/pull/685

**Pull request closed without merging**